opm/input/eclipse/EclipseState/Grid/FaceDir.cpp

```diff
@@ -100,8 +100,14 @@ namespace Opm {
 
         DirEnum FromIntersectionIndex(int idx)
         {
-            const std::vector<FaceDir::DirEnum> mapping =
-                {XMinus, XPlus, YMinus, YPlus, ZMinus, ZPlus};
+            static const std::vector<FaceDir::DirEnum> mapping{
```

Comment:
Is there a reason it needs to be a vector at all?  Could it not be a `std::array<>` instead, e.g.,
```C++
static constexpr auto mapping = std::array { XMinus, XPlus, ..., ZPlus, };
```

Reply:
no reason, changed.